install.sh

```diff
@@ -851,10 +859,47 @@ ${LOG_FILE} {
 EOF
 }
 
+create_runit_service_file() {
+    $SUDO mkdir -p ${FILE_K3S_SERVICE}/log
+    LOG_FILE=/var/log/${SYSTEM_NAME}.log
+    info "runit: Creating service file ${FILE_K3S_SERVICE}"
+    $SUDO tee ${FILE_K3S_SERVICE}/run >/dev/null << EOF
+#!/bin/sh 
+
+exec 2>&1; set -e
+
+rm -f /tmp/k3s.*
+
+set -o allexport
+if [ -f /etc/environment ]; then source /etc/environment; fi
+if [ -f ${FILE_K3S_ENV} ]; then source ${FILE_K3S_ENV}; fi
+set +o allexport
+
+modprobe -q loop || exit 1
+mountpoint -q /sys/fs/cgroup/systemd || {
+    mkdir -p /sys/fs/cgroup/systemd;
+    mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd;
```

Comment:
this is interesting, why is it necessary to spoof the systemd cgroups? We don't do this for openrc.

Reply:
I dont know if it is explicitly required for all the functions but afik in the case of docker, docker and runsvdir don't quite agree on what each signal means and there may be some incompatibilities, which is why the official docker-runit package on Artix also uses this approach.

Reply:
We don't use docker though, we use containerd. Does K3s work properly without this? It's not something I've ever seen before.

Reply:
My bad I should of worded it differently, I will test it without the systemd spoofing as I don't know if its actually required.